.github/workflows/ci.yml

```diff
@@ -66,6 +66,28 @@ jobs:
         bundle exec rake bench
       env:
         RAILS_VERSION: main
+  stories:
+    needs: lint
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Setup Ruby
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: 2.7
+    - uses: actions/cache@v2
+      with:
+        path: vendor/bundle
+        key: gems-build-stories-ruby-2.7-${{ hashFiles('**/Gemfile.lock') }}
+    - name: Build gem
+      run: |
+        gem install bundler:2.2.9
+        bundle config path vendor/bundle
+        bundle update
+        cd demo
+        bundle config path vendor/bundle
+        bundle update
+        bin/rails view_component_storybook:write_stories_json
```

Comment:
Does this return with a non-zero exit code if it fails? I think I've seen the process error but continue sometimes locally.

Reply:
Agreed ☝🏻 

I'd love to see a failed build on this branch showing that this works 😄

Reply:
well, you can see the current branch because it **is** failing 😄

Reply:
> but continue sometimes locally
that's because of overmind, the rake itself can be run once and returns non-zero https://github.com/primer/view_components/pull/312/checks?check_run_id=2017286033

Reply:
Could we confirm it fails on a bad story config as opposed to a bad import?

Reply:
yeah it fails!

Reply:
Oh, nice!

I get this locally and it carries on:

```
success open and validate gatsby-configs - 0.054s
doctocat  | ⠋ load plugins
storybook | ERR! SyntaxError: /Users/srt32/src/github.com/primer/view_components/demo/node_modules/.cache/storybook/dev-server/325c8f456729b912b0d2134054eb7448-782cbfe62df57bce0e5a00e62369408c: Unexpected end of JSON input
storybook | ERR!     at JSON.parse (<anonymous>)
storybook | ERR!     at /Users/srt32/src/github.com/primer/view_components/demo/node_modules/file-system-cache/node_modules/jsonfile/index.js:36:18
storybook | ERR!     at /Users/srt32/src/github.com/primer/view_components/demo/node_modules/graceful-fs/graceful-fs.js:123:16
storybook | ERR!     at FSReqCallback.readFileAfterClose [as oncomplete] (node:internal/fs/read_file_context:75:3)
storybook | ERR!  SyntaxError: /Users/srt32/src/github.com/primer/view_components/demo/node_modules/.cache/storybook/dev-server/325c8f456729b912b0d2134054eb7448-782cbfe62df57bce0e5a00e62369408c: Unexpected end of JSON input
storybook | ERR!     at JSON.parse (<anonymous>)
storybook | ERR!     at /Users/srt32/src/github.com/primer/view_components/demo/node_modules/file-system-cache/node_modules/jsonfile/index.js:36:18
storybook | ERR!     at /Users/srt32/src/github.com/primer/view_components/demo/node_modules/graceful-fs/graceful-fs.js:123:16
storybook | ERR!     at FSReqCallback.readFileAfterClose [as oncomplete] (node:internal/fs/read_file_context:75:3)
storybook |
storybook | WARN Broken build, fix the error above.
storybook | WARN You may need to refresh the browser.
```

Reply:
you should look for the `stories` process instead! `storybook` is what runs and serves the Storybook assets

Reply:
oh, that was my point, I think!

Are the problems we run into when we generate the stories or when we try to load the stories?

Reply:
oh, the problem is in the generation!